app/migration/parity_check/dynamic_request_content.rb

```diff
@@ -138,7 +138,7 @@ def participant_withdraw_payload(participant)
           type: "participant-withdraw",
           attributes: {
             reason: TrainingPeriod.withdrawal_reasons.values.map(&:dasherize).sample,
-            course_identifier: participant.api_ect_training_record_id.present? ? "ecf-induction" : "ecf-mentor",
+            course_identifier: course_identifier_for(participant),
```

Comment:
Can we make sure this PR is rebased/updated to use `course_identifier_for` for the defer request body as well, or @mooktakim's PR is rebased/updated with this depending on which gets merged first